app/js/components/MarginSelectorForm.jsx

```diff
@@ -0,0 +1,124 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {FormGroup, Label, InputGroup, InputGroupAddon, Alert} from 'reactstrap';
+import Input from 'react-validation/build/input';
+import '../../../node_modules/react-datetime/css/react-datetime.css';
+import '../../css/Form.scss';
+import {withNamespaces} from 'react-i18next';
+import {withRouter} from 'react-router-dom';
+import Form from 'react-validation/build/form';
+import {isInteger, required} from '../validators';
+
+class MarginSelectorForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      error: '',
+      isAbove: true,
+      marketError: '',
+      rate: 0
+    };
+  }
+
+  onRadioButtonChange = (e) => {
+    this.setState({isAbove: e.target.value === "above"});
+  }
+
+  onInputChange = (e) => {
+    this.setState({rate: e.target.value});
+  }
+
+  componentDidMount() {
+    // Redirecting to previous step
+    if(!this.props.fiat) this.props.history.push('/seller/fiat');
```

Comment:
we will need a real wizard here in future

Reply:
Indeed. I'll refactor this using the wizard component from the other PR